mailpoet/assets/js/src/automation/editor/components/header/more_menu.tsx

```diff
@@ -32,6 +38,18 @@ export function MoreMenu(): JSX.Element {
             />
           </MenuGroup>
           <MenuGroup>
+            <MenuItem
+              onClick={() => {
+                window.location.href = addQueryArgs(
+                  MailPoet.urls.automationAnalytics,
+                  {
+                    id: automation.id,
+                  },
+                );
+              }}
+            >
+              {__('Analytics', 'mailpoet')}
+            </MenuItem>
```

Comment:
Maybe for later: I think we should make this a filter so that the `mailpoet` integration can add analytics as analytics is not part of `core`.

Reply:
Yes, that's open and would be nice to have some in core.